tools/depends/target/libinput/Makefile

```diff
@@ -0,0 +1,37 @@
+include ../../Makefile.include
+DEPS=../../Makefile.include Makefile
+
+# lib name, version
+LIBNAME=libinput
+VERSION=1.8.4
```

Comment:
newer version available

Reply:
newer version uses meson

Reply:
if you ever want to upgrade to a newer version, we need meson, so we should do it correctly from the start

Reply:
feel free to add meson support